var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1202,7 +1349,8 @@ def deactivate(self, ext_pkg, view, **args):
                                         ))
 
     def add_files_to_view(self, view, merge_map):
-        bin_dir = self.spec.prefix.bin
+        bin_dir = self.spec.prefix.bin if os.name != 'nt'\
```

Comment:
..... because I forgot to turn of my `package.py` file name filter when I was grepping to change it.
This is my bad, I'll re run the replace without the filter.